auto_create.go

```diff
@@ -212,23 +246,25 @@ func (resController *ClusterWatcher) parseAutoCreateResourceInfo(unstructuredObj
 			resourceInfo.autoCreateName = toDomainName(resourceInfo.autoCreateName)
 		}
 	}
+
 	tmp, ok = annotations[AppAutoCreateKinds]
 	if !ok {
-		resourceInfo.autoCreateKinds = defaultKinds
+		resourceInfo.autoCreateKinds = defaultKinds			
 	} else {
 		autoCreateKindsStr, ok := tmp.(string)
 		if !ok {
 			if logger.IsEnabled(LogTypeError) {
 				logger.Log(CallerName(), LogTypeError, fmt.Sprintf("Metadata %s for resource %s/%s is not a string of comma separated kinds. Using default.", AppAutoCreateKinds, resourceInfo.resourceInfo.namespace, resourceInfo.resourceInfo.name))
 			}
-			resourceInfo.autoCreateKinds = defaultKinds
+			resourceInfo.autoCreateKinds = defaultKinds			
 		} else {
 			arrayOfString := stringToArrayOfAlphaNumeric(autoCreateKindsStr)
 			if len(arrayOfString) == 0 {
 				if logger.IsEnabled(LogTypeError) {
 					logger.Log(CallerName(), LogTypeError, fmt.Sprintf("Metadata %s for resource %s/%s does not contain comma separated kinds. Its current value is %s. Switching to default.", AppAutoCreateKinds, resourceInfo.resourceInfo.namespace, resourceInfo.resourceInfo.name, autoCreateKindsStr))
-				}
-				resourceInfo.autoCreateKinds = defaultKinds
+				}				
+				resourceInfo.autoCreateKinds = []groupKind{
+					groupKind{group: "apps", kind: resourceInfo.kind}}
```

Comment:
The group value "apps" will not work for all resource kinds.  It only works for "core" kinds for backward compatibility.  The group value should be obtained from resource's apiVersion field with this algorithm:

if apiVersion contains group name, use it
else use group name "core"

e.g. 
```
apiVersion: v1 ----> group name is "" 
apiVersion: extensions/v1beta1 ---> group name is "extensions" 
apiVersion: openliberty.io/v1beta1 ---> group name is "openliberty.io" 
```

Reply:
it is for debugging and forgot to clean up it.  It uses default component kinds in the existing code.